tests/playwright/sanity/includes/widgets/tabs.test.js

```diff
@@ -4,24 +4,13 @@ const WpAdminPage = require( '../../../pages/wp-admin-page.js' );
 test( 'Ensure the old tabs widget is telling deprecation warning message', async ( { page }, testInfo ) => {
 	// Arrange.
 	const wpAdmin = new WpAdminPage( page, testInfo );
-
-	await wpAdmin.setExperiments( {
-		container: true,
-		'nested-elements': true,
-	} );
-
-	const editor = await wpAdmin.useElementorCleanPost();
+	const editor = await wpAdmin.openNewPage();
```

Comment:
Why `openNewPage()` and not `useElementorCleanPost()`?

Reply:
it failed , the environment might be dirty

Reply:
Done